### ISSUE TITLE
No notification receive

### ISSUE THREAD

**Issue by victory7** [Minishlink/web-push-php-example#20]
I get in response "Message sent successfully for subscription" but I don't get any message!
I create my own VAPID but still, nothing changed.
What's going wrong in this?

**Comment by nomanaadma** [comment]
is it still there?

**Comment by deepakpalnitie** [comment]
I've enabled Push Notifications, but after pressing 'Send a Push Notification' nothing happens. Everything remains silent, nothing on page or on console.

**Comment by miok73** [comment]
I dont' know why but in localhost everything works well; I've loaded all folders in cloud and nothing happen ... Has someone a idea please?

**Comment by levizwannah** [comment]
This is not a solution for everyone, but make sure to turn on notifications for chrome from your device settings (Windows, Mac, etc)

**Comment by Tarek-Adra** [comment]
same here

**Comment by robinmalik** [comment]
> This is not a solution for everyone, but make sure to turn on notifications for chrome from your device settings (Windows, Mac, etc)

At first I was getting an error 500 when clicking `send a push notification` but running `composer update` fixed this, but still no notification. The fix for that was as above, under 'Notifications & actions' on Windows 10. Thanks @levizwannah

**Comment by Tarek-Adra** [comment]
For me it was windows blocking it, I was enabling "Focus assist" and set to "priority only" after turn it off it's working perfectly :)

**Comment by sahilmore-git** [comment]
I am also not getting any notifications. Just the request is 200 success and says subscription is successful. 
![image](https://user-images.githubusercontent.com/46355027/131735460-390fdb57-61ff-4c2c-867d-65fcbdf3d41f.png)

**Comment by sahilmore-git** [comment]
Any solutions?? @Tarek-Adra @robinmalik @levizwannah @miok73 @deepakpalnitie @nomanaadma @victory7

**Comment by nomanaadma** [comment]
@sahilmore-git
It looks like you are using a firefox browser make sure to enable notification from the topbar according to the new firefox policy
and is it sending notifications in chrome?

**Comment by sahilmore-git** [comment]
@nomanaadma  Yes I have enabled notifications in firefox and it is also not working in chrome :(

**Comment by xitude** [comment]
> I dont' know why but in localhost everything works well; I've loaded all folders in cloud and nothing happen ... Has someone a idea please?

Works perfectly in chrome and the second I move to production it fails me, do we need unique VAPID stuff? Like are they attached to our first used domain?

**Comment by xh8039** [comment]
我启用了推送通知，但在按下“发送推送通知”后没有任何反应。一切都保持沉默，页面或控制台上没有任何内容
